src/shared/modules/commands/commandsDuck.test.js

```diff
@@ -43,6 +43,10 @@ import {
   replace as replaceSettings
 } from 'shared/modules/settings/settingsDuck'
 import { cleanCommand, getInterpreter } from 'services/commandUtils'
+import bolt from 'services/bolt/bolt'
```

Comment:
Bolt should not be needed in this file since we're not testing the new format, right?

Reply:
I've just added a test in the latest commit that requires this.